share/completion/cal

```diff
@@ -0,0 +1,39 @@
+# (C) 2025 unrealapex
+
+# Completion script for cal
+
+function completion/cal {
+
+        typeset OPTIONS ARGOPT PREFIX
+        OPTIONS=( #>#
+        "l; turn off highlighting of today"
+        "J; display Julian Calendar"
+        "e; display date of Easter"
+        "j; display Julian days"
+        "M; display monday as the first day of the week"
+        "m; display the specified *month*"
```

Comment:
How should I denote that this option needs an argument?

Reply:
Append a colon to the option character.

```suggestion
        "m:; display the specified *month*"
```